### DISCUSSION TITLE
Export multiple notes  as a single combined PDF

### DISCUSSION THREAD

**Discussion by T-Bolt94** [windingwind/zotero-better-notes#1203] [Q&A]
Is it possible to export multiple notes under one parent item as a single PDF? i.e. merge multiple notes into one PDF?

**Comment by windingwind** [comment]
Add all note links to one note, and then export the note with embedded note link content.

**Reply by T-Bolt94** [reply]
Thanks for this suggestion but when I do this and select PDF with the embedded link content it just creates a new Zotero note? Am I doing something wrong?

**Reply by T-Bolt94** [reply]
![Screenshot 2024-11-18 at 19 10 14](https://github.com/user-attachments/assets/5f34bbcc-cd1b-430a-a295-03b18784a813)

**Reply by T-Bolt94** [reply]
It will not embed the linked note if it has images within it which I need it to do. Is there a work around for this?

**Reply by windingwind** [reply]
update plugin

**Reply by T-Bolt94** [reply]
Excellent!! Thanks

**Comment by T-Bolt94** [comment]
Is there a way to add the notes parent item title to the builtin template? I have tried alot but coding isn't my thing at all. 
What I want to have is 

The "link embedded notes parent item title" - Then the embedded note below
${{
  return await Zotero.BetterNotes.api.convert.link2html(link, {noteItem, dryRun: _env.dryRun});
}}$
Then a 2-3px thick horizontal black line under

Sorry for all the questions...just started using BetterNotes and love it but trying to learn how to code for it!
